components/messenger/transports.rst

```diff
@@ -0,0 +1,225 @@
+.. index::
+    single: Messenger; Transports
+
+Transports
+==========
+
+To send and receive messages via message brokers, the Messenger component has
+"transports". Transports are responsible for routing messages to and from
+the message brokers.
+
+Every transport is configurable using a DSN. This DSN allows you to chose the
+transport layer as well as configure it.
+
+AMQP
+----
+
+The most famous message broker protocol is probably AMQP, which is most
+commonly implemented with RabbitMQ. The Messenger component has built-in
+support for AMQP.
+
+How it works?
+~~~~~~~~~~~~~
+
+A DSN protocol that uses ``amqp`` is recognized and sent to the built-in AMQP transport.
+For example::
```

Comment:
Could we remove this „for example“ and instead add another sentence to the paragraph before and end with „::“ ?

Reply:
@OskarStark sorry, not clearly understand for what we need to add instead of removed one?

Reply:
Let’s write:

... sent to the built-in AMQP transport::

And remove the “for example...” line completely

Reply:
thanks!